closure/goog/math/integer.js

```diff
@@ -566,6 +566,60 @@ goog.math.Integer.carry16_ = function(bits, index) {
   }
 };
 
+/**
+ * Returns this Integer divided by the given one.
+ * NOTE: Both this and the given Integer MUST be positive.
+ * @param {goog.math.Integer} other The Integer to divide this by.
```

Comment:
But if I'm reading the code correctly, the `divide` method will crash on its first line (calling `.isZero`) if its argument is null, is that correct? So if `slowDivide` is only called by `divide` its argument should never be null?

And what does coding defensively mean here? What should this method do if it receives a null argument?